ISM43362/ISM43362.cpp

```diff
@@ -175,18 +175,6 @@ bool ISM43362::check_response(void)
         return false;
     }
 
-    /*  Inventek module do stuffing / padding of data with 0x15,
```

Comment:
Does it mean that 0x15 padding has been replaced by 0x3E padding in more recent Inventek firmware version ? 
In the specification I found, we still see references to 0x15 padding ?

Reply:
@LMESTM
The last useful character is ">" -> ASCII (0x3E).
So, I check the presence of ">" and I flush.
But after the ">" there are useless characters (0x15) which were previously removed
Please, is it possible to continue this conversation in PR (#90) which is up to date now.